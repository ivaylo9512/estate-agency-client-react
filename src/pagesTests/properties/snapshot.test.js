import { shallow } from 'enzyme';
import Properties from 'pages/properties';

describe('properties snapshot tests', () => {
    const createWrapper = () => shallow(
        <Properties />
    )

    it('should match snapshot', () => {
        const wrapper = createWrapper();
    
        expect(wrapper).toMatchSnapshot();
    })
})