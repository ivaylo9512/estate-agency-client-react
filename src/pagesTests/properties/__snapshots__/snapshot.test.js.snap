// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`properties snapshot tests should match snapshot 1`] = `
<section>
  <PropertiesSearchForm />
  <PropertiesList />
  <PropertiesPagination
    getProperties={[Function]}
    selector={[Function]}
    setCurrentProperties={[Function]}
  />
</section>
`;
